tests/utils/tracked_container.py

```diff
@@ -54,9 +54,9 @@ def run_detached(self, **kwargs: Any) -> None:
         )
         LOGGER.info(f"Container {self.container.name} created")
 
-    def get_logs(self) -> str:
+    def get_logs(self, **kwargs: Any) -> str:
```

Comment:
Let's add `stdout` and `stderr` params with their default values, to make it easy for IDEs give suggestions and show that we only care about these params.

Reply:
Good call, done.